docs/source/tutorial.md

```diff
@@ -0,0 +1,43 @@
+# Tutorial
+
+This page covers a brief tutorial on how to use pycytominer-transform.
+
+## Data Source Locations
+
+Data sources may be provided to pycytominer-transform using local filepaths or object-storage (for example, "AWS S3", "GCP Cloud Storage", "Azure Storage") filepaths (via [cloudpathlib](https://cloudpathlib.drivendata.org/~latest/)).
+Object storage paths which are public-facing and do not require authentication may be enabled using `convert(..., no_sign_request=True)`.
```

Comment:
I'm not sure what this step means. What does the convert command do? Where is this invoked?

Reply:
Thank you for this comment - I think I need to reword this a bit more. I intended to make sure someone reading the docs would understand they may experience an error with public-facing/unauthenticated object storage access unless they specify the appropriate cloudpathlib args (`no_sign_request=True` is equivalent to `awscli s3 ... --no-sign-request`).